waspc/examples/todo-typescript/.wasp/out/sdk/wasp/package.json

```diff
@@ -50,7 +50,7 @@
       "./server/actions": "./dist/server/actions/index.js",
       "./server/queries": "./dist/server/queries/index.js",
       "./dbSeed/types": "./dist/dbSeed/types.js",
-      "./jobs/*": "./dist/jobs/*",
+      "./jobs/*": "./dist/jobs/*.js",
```

Comment:
We want all public imports to be extensionless, as it works better with both the user code and `ModuleImportPath` from `JsImport`.